Source/Core/DolphinQt/Debugger/BranchWatchDialog.cpp

```diff
@@ -207,7 +207,6 @@ BranchWatchDialog::BranchWatchDialog(Core::System& system, Core::BranchWatch& br
 {
   setWindowTitle(tr("Branch Watch Tool"));
   setWindowFlags((windowFlags() | Qt::WindowMinMaxButtonsHint) & ~Qt::WindowContextHelpButtonHint);
-  SetQWidgetWindowDecorations(this);
```

Comment:
Any reason this moved?

Reply:
It's a reversion, see the now removed FIXME comment. This is where the `SetQWidgetWindowDecorations` call used to be before the Branch Watch Tool replaced the Code Diff Tool; putting it just before a `show` or `exec` call mirrors most other uses of the function across the codebase. To be honest, I don't actually know what difference it makes.